requirements.txt

```diff
@@ -4,3 +4,4 @@ element-lab==0.1.0b0
 element-animal==0.1.0b0
 element-session==0.1.0b0
 ipykernel==6.0.1
+jupytext==1.13.7
```

Comment:
Does a user need to install jupytext when they install the workflow? Will they get an error otherwise? I'll read more abotu how jupytext works behind the scenes. I thought it functioned like a package install for jupyter itself that launched when saving

Reply:
Jupytext should only be required for syncing the paired notebooks/scripts.

Reply:
By including it in `requirements.txt`, I think that would install it for average users, while this is more of a dev dependency. A more sophisticated installation would separate user and dev requirements ([example here](https://github.com/numba/numba/blob/main/setup.py), though I don't fully understand it) - something to consider in the future